source/whats-new.txt

```diff
@@ -56,6 +57,45 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-6.5:
+
+What's New in 6.5
+-----------------
+
+The {+driver-short+} v6.5 release includes the following features:
+
+- Moves ``ObjectId`` generation to the ``pkFactory`` class for bulk write
+  operations.
```

Comment:
```suggestion
- Bulk write operations generate document ids using the ``pkFactory``
```

Reply:
Thanks, I'll change "ObjectID generation" to "generate document ids".
To maintain the style, the bullet point needs to start with a present tense verb. I'll change it to the following (feel free to suggest something else):

"Updates bulk write operations to use the pkFactory class for document id generation."

Reply:
yup, that works too.  thanks!